pre/main.ipynb

```diff
@@ -25,46 +25,94 @@
   },
```

Comment:
_Commented on notebook `pre/main.ipynb` Cell 18 Line 51_
```python
            dc["services"][deployment_id] = entry

        # IDE Container definition
        ide_entry = {}
        ide_entry["entrypoint"] = "/start-ide.sh"
        ide_entry["image"] = "dev_" + deployment_id
        ide_entry["volumes"] = entry.get("volumes", [])
        ide_entry["network_mode"] = "host"
```
Can we specify ports instead and pass an explicit port to jupyterlab?

_Respond and view the context [here](https://gitnotebooks.com/bitswan-space/BitSwan/pull/13#comment-1142)._